docs/spec/json-web-signatures.md

```diff
@@ -19,25 +19,48 @@ Here is an example of a decoded JWS header:
 }
 ```
 
+::: warning
+  It is recommended that `kid` be a DID URL. If it is not, method implementers might need to rely on additional context to uniquely identify the associated verificationMethod. 
+:::
+
 ### Verifying
 
-If `kid` is a DID URL, the verification key MAY be resolved as follows:
+Regardless of which proof purpose a verificationMethod is associated with, the process of verifying a JWS linked to a DID is the same.
 
-1. Resolve the `kid` value using a resolver.
-2. iterate the `operations` and `recovery` verificationMethods, until a verificationMethod with `id` equal to `kid` is found.
-3. A [Recover Operation](https://identity.foundation/sidetree/spec/#recover) is only valid when signed by a key listed in `recovery`. Other operations are only valid when signed with keys listed in `operations`. If the key is not present in the expected collection, verification fails, and the operation is considered invalid. If the key is present in the expected collection proceed.
-3. Convert the discovered verificationMethod to JWK if necessary.
-4. Perform [JWS Verification](https://tools.ietf.org/html/rfc7515#section-5.2) using the JWK.
-5. The operation is considered valid if key material was present in the correct collection and the signature is valid.
+The JWS header is parsed and a `kid` is extracted.
 
+1. Iterate the verificationMethods, until a verificationMethod with `id` equal to `kid` is found.
+2. Convert the discovered verificationMethod to JWK if necessary.
+3. Perform [JWS Verification](https://tools.ietf.org/html/rfc7515#section-5.2) using the JWK.
 
-If `kid` is not a DID URL, the verification key MAY be resolved as follows:
+#### Operation Verification
 
-1. Iterate an internal representation of keys registered for use with the DID until a jwk with `kid` equal to `kid` is found.
-2. Ensure that the `usage` property of the internal key represenation is `recovery` if the operation is a recovery operation or `ops` if the operation is not Recovery or Update.
-3. Perform [JWS Verification](https://tools.ietf.org/html/rfc7515#section-5.2) using the JWK.
-4. The operation is considered valid if key material was present with the correct usage and the signature is valid.
+Sidetree operations are considerd valid when the JWS can be verified, and where the key used is associated with the correct proof purpose.
```

Comment:
thanks, it should say verifcationMethod instead of key... we could theoretically use `ethereumAddress` or `bitcoinAddress` and EC Recover to verify sidetree operations.